R/uvhydrograph-fnc.R

```diff
@@ -157,11 +165,11 @@ combineLims<- function(lims1, lims2, ...) {
   return(list(xlim = xlim, ylim = ylim))
 }
 
-add_uvhydro_axes <- function(lims, ylog = TRUE, ylab){
+createPlot <- function(lims, ylog = TRUE, ylab) {
```

Comment:
kind of a generic name given creates a specific type of plot? or is this meant to be a generic plot set up? If the latter, should probably be exported and documented?